test/com/google/enterprise/adaptor/database/JdbcFixture.java

```diff
@@ -71,7 +112,23 @@ public static void dropAllObjects() throws SQLException {
         throw new AssertionError(e);
       }
     }
-    executeUpdate("drop all objects");
+    if (DATABASE.equalsIgnoreCase("h2")) {
```

Comment:
We may want an enum for these, just to have a canonical reference. There's some code elsewhere, I think in the DB connector, to get case-insensitive enum values from a string.